From: Simon McVittie <smcv@debian.org>
Date: Fri, 14 Aug 2020 12:56:47 +0100
Subject: build: Add missing pkg-config Requires.private dependencies

Otherwise, a trivial executable that includes <act/act.h>, built with
$(pkg-config --cflags --libs accountsservice), will fail to compile
when we cannot include headers like <glib-object.h>.

Forwarded: https://gitlab.freedesktop.org/accountsservice/accountsservice/-/merge_requests/62
Applied-upstream: 0.6.56, commit:0376129b92490789ca337d8cd9f6b3bcbd00e416
---
 src/libaccountsservice/meson.build | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/libaccountsservice/meson.build b/src/libaccountsservice/meson.build
index 4e134db..a5b5b1f 100644
--- a/src/libaccountsservice/meson.build
+++ b/src/libaccountsservice/meson.build
@@ -92,6 +92,12 @@ pkg.generate(
     'datadir=${prefix}/' + get_option('datadir'),
     'interfacesdir=${datadir}/accountsservice/interfaces',
   ],
+  requires_private : [
+    'gio-2.0',
+    'glib-2.0',
+    'gobject-2.0',
+    'polkit-gobject-1',
+  ],
 )
 
 if get_option('introspection')
